server/api_channel.go

```diff
@@ -103,6 +103,17 @@ func (p *Plugin) handleSince(c *gin.Context) {
 		return
 	}
 
+	p.track(evSummarizeUnreadMessages, map[string]interface{}{
+		"channel_id": channel.Id,
+		"user_id":    user.Id,
+		"since":      data.Since,
+		"type":       promptPreset,
+		"feature": map[string]string{
```

Comment:
Seems strange to add this here? The SKU is implied and all our events should be tagged with AI.

Reply:
That comes from the ticket here: https://mattermost.atlassian.net/browse/MM-58196

Reply:
@esethna What do you think?

Reply:
From the jira ticked I think this is consider the right way to do it.